examples/vanilla/tooltips.html

```diff
@@ -0,0 +1,246 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Media Chrome Tooltips</title>
+    <script type="module" src="../../dist/index.js"></script>
+    <style>
+      body {
+        background: #eee;
+        padding: 50px;
+      }
+      
+      .examples {
+        margin-top: 20px;
+      }
+      
+      media-controller {
+        max-width: 300px;
+      }
+      
+      media-control-bar {
+        margin: 10px;
+      }
+
+      code {
+        color: maroon;
+      }
+
+      .custom-btn {
+        position: relative;
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+        background: lightblue;
+        border: 2px solid #333;
+        color: #333;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        font-size: 24px;
+        font-weight: bold;
+        cursor: pointer;
+      }
+      .custom-btn media-tooltip {
+        opacity: 0;
+        transition: opacity .3s;
+      }
+      .custom-btn:hover media-tooltip {
+        opacity: 1;
+      }
+    </style>
+  </head>
+  <body>     
+    <main>
+      <h1>Tooltips</h1>
+
+      <h2>All buttons</h2>
+
+      <media-controller style="--media-tooltip-container-margin: 10px; min-width: 700px;">
+        <video
+          slot="media"
+          src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/high.mp4"
+          muted
+        ></video>
+        <media-control-bar style="display: flexbox; gap: 5px;">
+          <media-airplay-button></media-airplay-button>
+          <media-audio-track-menu-button></media-audio-track-menu-button>
+          <media-captions-menu-button></media-captions-menu-button>
+          <media-captions-button></media-captions-button>
+          <media-cast-button></media-cast-button>
+          <media-fullscreen-button></media-fullscreen-button>
+          <media-mute-button></media-mute-button>
+          <media-pip-button></media-pip-button>
+          <media-play-button></media-play-button>
+          <media-playback-rate-button></media-playback-rate-button>
+          <media-playback-rate-menu-button></media-playback-rate-menu-button>
+          <media-rendition-menu-button></media-rendition-menu-button>
+          <media-seek-backward-button></media-seek-backward-button>
+          <media-seek-forward-button></media-seek-forward-button>
+          <media-settings-menu-button></media-settings-menu-button>
```

Comment:
could the menu be added in the example so the interaction between tooltip and menu showing can be tested?